### PR TITLE
Remove prop mutation

### LINKED ISSUE #1021: Do not mutate prop locally
https://github.com/aeternity/superhero-ui/blob/c3e4b97b07ac10c66c8b737d49c79fcf8a4a0179/src/components/VoteCard.vue#L86

You can read more about this in
https://vuejs.org/v2/guide/migration.html#Prop-Mutation-deprecated

### PR DESCRIPTION
**Pull request by AtanasKrondev**
Fixes #1021

### CODE REVIEW

**Comment by thepiwo** [comment]
@kenodressel @AtanasKrondev whats the state of this?

**Comment by AtanasKrondev** [comment]
> @kenodressel @AtanasKrondev whats the state of this?

I propose merging

File: src/components/VoteCard.vue [hunk 1]
```diff
@@ -178,11 +178,13 @@ export default {
     loading: false,
     progressMessage: '',
     description: '',
+    stakeAmount: 0,
   }),
   computed: {
     ...mapState(['tokenInfo']),
   },
   async mounted() {
+    this.stakeAmount = +this.vote.stakeAmount;
```
Comment (kenodressel) [inline src/components/VoteCard.vue:187]: why the `+` here?

Comment (AtanasKrondev) [reply src/components/VoteCard.vue:187]: Probably I had some precautions with shifting decimals which I can not remember.🤷🏻‍♂️
I just tested it and it's working just fine as a string, so indeed the ``+`` is unnecessary

**Review by kenodressel — APPROVED** [review]
lets wait with the merge until tips from db is released

**Pull request merged**